src/manifest.jl

```diff
@@ -284,8 +284,9 @@ function write_manifest(env::EnvCache)
 end
 function write_manifest(manifest::Manifest, manifest_file::AbstractString)
     if manifest.manifest_format.major == 1
-        @warn """The active manifest file at `$(manifest_file)` has an old format that is being maintained.
-            To update to the new format run `Pkg.upgrade_manifest()` which will upgrade the format without re-resolving.""" maxlog = 1 _id = Symbol(manifest_file)
+        @warn """The active manifest file has an old format that does not record the julia version. The dependencies
```

Comment:
Perhaps allow an override?
```suggestion
        do_warn = tryparse(Int, get(ENV, "JULIA_PKG_OLD_MANIFEST_WARN", "0")) == 1
        do_warn && @warn """The active manifest file has an old format that does not record the julia version. The dependencies
```